packages/fx-core/src/plugins/resource/frontend/index.ts

```diff
@@ -81,6 +81,13 @@ export class FrontendPlugin implements Plugin {
     );
   }
 
+  public async generateArmTemplates(ctx: PluginContext): Promise<TeamsFxResult> {
+    FrontendPlugin.setContext(ctx);
+    return this.runWithErrorHandling(ctx, TelemetryEvent.Deploy, () =>
```

Comment:
I suppose yes?

Reply:
In which stage will generateArmTemplates() be called, provision or scaffolding or a new stage? I don't think it is in deployment stage

Reply:
generateArmTemplates() is called during provision stage. Do you think it the telemetryEvent here should be `Scaffold` or `GenerateArmTemplates`?

Reply:
I think it's called in scaffold stage. Is it typo?

Reply:
@eriolchan , This telemetry event name is changed to `TelemetryEvent.GenerateArmTemplates`. Using a new event name will be better since this function can be called in different stages. Thx.